templates/common/_base/units/ovs-notification.yaml

```diff
@@ -0,0 +1,20 @@
+name: ovs-notification.service
+enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else if eq .NetworkType "OpenShiftSDN"}}true{{else}}false{{end}}
+contents: |
+  [Unit]
+  Description=Creates a file to let the OVS pod know OVS is running on systemd.
+  # Don't write the notification before OVS starts
+  Requires=openvswitch.service
+  Wants=NetworkManager-wait-online.service
+  After=NetworkManager-wait-online.service openvswitch.service
+  Before=network-online.target kubelet.service crio.service node-valid-hostname.service
+
+  [Service]
+  # Need oneshot to delay kubelet
+  Type=oneshot
+  ExecStart=/usr/bin/touch /var/run/openvswitch/ovs-notification
```

Comment:
Might be worth a comment that we put this in /var/run/ (which we assume to be a tmpfs) so that it gets deleted automatically on restart.

Reply:
done